packages/wdio-protocols/index.d.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference types="node"/>
```

Comment:
let's add a typing test for this one. At least very superficial one to make sure types can be imported and not conflicting with existing ones

Reply:
I added some protocols typing checks